RxNimble.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = "9.0"
   s.source       = { :git => "https://github.com/ashfurrow/RxNimble.git", :tag => s.version }
   s.source_files  = "Source/**/*.swift"
-  s.dependency "Nimble", "~> 6.0"
+  s.dependency "Nimble", "~> 7.0"
```

Comment:
Since this is a major bump to a dependency, we'll need to upgrade RxNimble to 3.0.0 instead of 2.0.1.